src/test/java/org/humancellatlas/ingest/schemas/SchemaTest.java

```diff
@@ -0,0 +1,77 @@
+package org.humancellatlas.ingest.schemas;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import org.humancellatlas.ingest.schemas.schemascraper.SchemaScraper;
+import org.humancellatlas.ingest.schemas.schemascraper.impl.SchemaScraperImpl;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.File;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collection;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+
+/**
+ * Created by rolando on 19/04/2018.
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest
```

Comment:
Use only a subset of the Spring configuration by defining a minimal Spring configuration. Limit the scope of test to either just scraping or the associated service. This test tries to configure stuff like Mongo DB, which isn't necessary to get this passing, and which slows down the test.